src/components/Stories/StoryContent.js

```diff
@@ -124,7 +126,11 @@ const StoryContent = ({
               )}
               style={{
                 '--width':
-                  active === idx ? (isPaused ? `${width}px` : '100%') : 0
+                  active === idx
+                    ? isPaused
```

Comment:
can it be simplified?

Reply:
tried a lot, but no(
I'll play here again later